enterprise/backup/src/test/java/org/neo4j/backup/impl/BackupProtocolServiceIT.java

```diff
@@ -789,12 +794,13 @@ public void shouldHandleBackupWhenLogFilesHaveBeenDeleted() throws Exception
     private GraphDatabaseAPI deleteLogFilesAndRestart()
             throws IOException
     {
-        final FileFilter logFileFilter = pathname -> pathname.getName().contains( "logical" );
+        final Predicate<Path> logFileFilter = p -> p.getFileName().toString().contains( "logical" );
```

Comment:
would we delete any "logical" file? shouldn't it use our log files abstraction instead?